ehrapy/preprocessing/_normalization.py

```diff
@@ -3,6 +3,7 @@
 from typing import TYPE_CHECKING
 
 import numpy as np
+from lamin_utils import logger as logg
```

Comment:
right?

Reply:
Yes and no. I do not shorten it to `logg` so just use `logger` directly, please